.github/workflows/functional.yaml

```diff
@@ -0,0 +1,50 @@
+name: "Functional"
+
+env:
+  terraform_version: "1.2.8"
+  C7N_FUNCTIONAL: true
+  C7N_TEST_RUN: true
+
+on:
+  schedule:
+    # UTC 6 AM aka 11 PM PST, 2 AM EST, everyday
+    - cron: "0 6 * * *"
+
+jobs:
+  AWS:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      checks: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4.2.0
+        with:
+          python-version: "3.8"
+
+      - uses: aws-actions/configure-aws-credentials@v1.7.0
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
+          aws-region: us-east-1
+
+      - name: Install Terraform
+        run: |
```

Comment:
nit: switch out for setup-terraform action from hashicorp, and move mkdir -p to dep install.

Reply:
done

Reply:
And example working here: https://github.com/thisisshi/cloud-custodian/runs/8214994814?check_suite_focus=true